### PR TITLE
Monda: Version 2.200 added



### PR DESCRIPTION
**Pull request by emmamarichal**
Taken from the upstream repo https://github.com/googlefonts/mondaFont at commit https://github.com/googlefonts/mondaFont/commit/13ca3d8bce80e715a736026c043f33e5c0bb5b83.
## PR Checklist:

- [x] Family categorization tags collected from the type design team with the Categories Form
- [x] Minisite_url definition in the METADATA.pb file for commissioned projects
- [x] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [x] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [x] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

File: ofl/monda/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -1,10 +1,18 @@
 <p>
-The Monda font family is a <em>libre</em> font family. 
-It has been designed to be used freely across the internet by web browsers on desktop computers, laptops and mobile devices.
+ The Monda font family is a
+ <em>
```
Comment (vv-monsalve) [inline ofl/monda/DESCRIPTION.en_us.html:3]: Please review the HTML formatting.

**Review by vv-monsalve — DISMISSED** [review]

**Comment by vv-monsalve** [comment]
@emmamarichal It seems the is a conflict between the static and VF fonts. See [build](https://github.com/google/fonts/actions/runs/8662257535/job/23773999267?pr=7549#step:6:149)

**Comment by emmamarichal** [comment]
@vv-monsalve I have the feeling it's a conflict with the old versions, since it's a VF replacement. Maybe it's something related to the new packager? But I see it packages the right file though.
I don't have any issues locally, here you can find the QA I exported. We can talk about this in the next pipeline meeting if Simon is here!

[Monda_QA.zip](https://github.com/google/fonts/files/15007863/Monda_QA.zip)

File: ofl/monda/DESCRIPTION.en_us.html (hunk 1 above)
Comment (vv-monsalve) [reply ofl/monda/DESCRIPTION.en_us.html]: Solved.

**Comment by vv-monsalve** [comment]
> I have the feeling it's a conflict with the old versions, since it's a VF replacement. Maybe it's something related to the new packager?

From our last tool meeting I think so. Perhaps given you updated the path to the VF on the metadata file before running it (AFAIR).

> But I see it packages the right file though.

I re-download the qa from the CI run and this is what happens with it
Screen Shot 2024-04-17 at 12 32 46

As discussed on the meeting today, we better wait to review the reported error from the CI build with Marc next week. 

Alternatively you could try to update the PR from packager (e.g. from a release with clean new paths to fonts)

**Comment by emmamarichal** [comment]
Ah, I maybe have an idea. This was a case a bit special, since it's a VF replacement. I edited the metadata.pb that already exists, to change the path to /variable/monda[wght] instead of /ttf/monda-regular.ttf etc.

But I forgot to change the fonts infos in metadata. There are still on bold and regular. Let me solve that today!
Screenshot 2024-04-18 at 11 05 32

**Comment by emmamarichal** [comment]
@vv-monsalve, I can't unblock the PR because of the "changes requested" :/

**Pull request merged**